v21.2/deploy-app-netlify.md

```diff
@@ -0,0 +1,227 @@
+
+The sample app used in this tutorial simulates a gaming leaderboard. The [Netlify functions](https://www.netlify.com/products/functions/) used for the app are written in TypeScript. The functions use [Prisma](https://www.prisma.io/) to connect to CockroachDB. The app's frontend, also written in TypeScript, uses React, bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+The source code for the completed app is available on GitHub at [https://github.com/cockroachdb/cockroachdb-typescript](https://github.com/cockroachdb/cockroachdb-typescript).
+
+{{site.data.alerts.callout_success}}
+See the blog post that inspired this tutorial: [How to build a Complete Webapp with React, TypeScript & CockroachDB](https://www.cockroachlabs.com/blog/react-typescript-cockroachdb-sample-app/#deploy-the-application-to-netlify).
+{{site.data.alerts.end}}
+
+## Prerequisites
+
+Before starting the tutorial, do the following:
+
+1. Create a [{{ site.data.products.db }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account.
+
+1. Create a Starter [Netlify](https://app.netlify.com/signup) account. You can do this with your GitHub login credentials.
+
+1. Install the [`cockroach` binary](install-cockroachdb.html). You'll use the [`cockroach sql` CLI](cockroach-sql.html) to initialize the database schema for the app.
+
+## Step 1. Create a {{ site.data.products.serverless }} cluster
+
+{% include cockroachcloud/quickstart/create-a-free-cluster.md %}
+
+<a name="connection-string"></a>
+
+After the cluster is created, the **Connection info** window appears. Click the **Connection string** tab and copy the connection string to a secure location. You will use this connection string to connect to CockroachDB later in the tutorial.
+
+{{site.data.alerts.callout_info}}
+The connection string is pre-populated with your username, cluster name, and other details, including your password. Your password, in particular, will be provided only once. Save it in a secure place (we recommend a password manager) to connect to your cluster in the future. If you forget your password, you can reset it by going to the [**SQL Users** page](../cockroachcloud/user-authorization.html).
+{{site.data.alerts.end}}
+
+## Step 2. Get the code
+
+Clone the code's GitHub repo:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+$ git clone git@github.com:cockroachdb/cockroachdb-typescript.git
+~~~
+
+The project has the following directory structure:
+
+~~~
+├── LICENSE.md
+├── README.md
+├── database
+│   └── schema.sql
+├── netlify
+│   └── functions
+│       ├── addScore.ts
+│       ├── getPlayers.ts
+│       └── getScores.ts
+├── package-lock.json
+├── package.json
+├── prisma
+│   └── schema.prisma
+├── public
+├── src
+└── tsconfig.json
+~~~
+
+The `database/schema.sql` file specifies the database schema for the application:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
```

Comment:
Nit: For me this was too much code before getting to interactive steps. I think it would be reasonable to introduce this file in step three, for example.